styles/vars.scss

```diff
@@ -422,14 +433,20 @@ body {
 
   // Dark Grid
   &.xh-dark {
-    --xh-grid-bg-hover: var(--grid-bg-hover, hsl(200, 50%, 17%));
+    --xh-grid-bg-hover: var(--grid-bg-hover, hsl(200, 50%, 22%));
     --xh-grid-bg-odd: var(--grid-bg-odd, hsl(200, 26%, 11%));
     --xh-grid-border-color: var(--grid-border-color, hsl(186, 11%, 18%));
     --xh-grid-cell-change-bg-highlight: var(--grid-cell-bg-highlight, #{mc-trans('orange', '800', 0.8)});
     --xh-grid-group-bg: var(--grid-group-bg, #{mc('blue-grey', '800')});
     --xh-grid-group-border-color: var(--grid-group-border-color, #{mc('blue-grey', '700')});
-    --xh-grid-tree-group-bg: var(--grid-tree-group-bg, hsl(200, 50%, 12%));
+    --xh-grid-tree-group-bg: var(--grid-tree-group-bg, hsl(200, 50%, 16%));
     --xh-grid-total-row-bg: var(--grid-total-row-bg, var(--xh-grid-tree-group-bg));
+    --xh-grid-tree-group-color-level-0: var(--grid-tree-group-color-level-0, hsl(205, 65%, 15%));
```

Comment:
These hsl colors look better than what I could create using Material colors w/ a darken function. Could take a second pass at using material colors for consistency with light theme

Reply:
I'm not especially loyal to the material colors, especially in dark mode.  (I wonder if there is a different set of palettes they make available specifically for dark themes?)

In some ways the material design color function ends up obscuring the underlying color / makes it a bit more difficult to reason about for a dev who wants to find some HSL variant or generally re-use the color w/o necessarily importing or using the mc functions.  We're clearly in an inconsistent state at the moment and I am not proposing we try to resolve everything one way or another - just saying I wouldn't want use of material colors to compromise the end result if we are happier with HSL.

Reply:
Updated to use HSL colors in light theme